kernel/scripts/deploy.js

```diff
@@ -26,8 +28,9 @@ async function deploy() {
   const developerFraction = 10;
   const zepCore = await ZepCore.new(newVersionCost, developerFraction, { from: zeppelin });
   console.log(" ZepCore: ", zepCore.address);
+  const registry = await Registry.new({ from: zeppelin });
   const factory = await UpgradeabilityProxyFactory.new();
-  const controller = await KernelProxyController.new(zepCore.address, factory.address);
+  const controller = await ProjectController.new(projectName, registry.address, factory.address, zepCore.address, { from: zeppelin });
```

Comment:
Not sure if this should be part of the deploy script now, but since this script is just for testing purposes, I'd merge anyway and then refactor